nuitka/utils/SharedLibraries.py

```diff
@@ -368,21 +385,21 @@ def _getSharedLibraryRPATHDarwin(filename):
                 if str is not bytes:
                     result = result.decode("utf8")
 
-                return result
+                rpaths.append(result)
```

Comment:
It's possible to have multiple `LC_RPATH` load commands (try `cc -shared -o blah -Wl,-rpath,a,-rpath,b; otool -L blah`). Since I had an excuse to make these functions return a list instead of a single value, I figured I'd make this change at the same time. I can submit it separately if we choose not to use this approach.

Reply:
Ah yes, that's just that I never saw a Python that does that.